src/Components/Label/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import './index.css';
+
+export default class LabelTitle extends Component {
+    render() {
+
+        const arrowImg = "http://via.placeholder.com/18x22";
+
+        return (
+            <div className="labelTitle">
+                <p className="labelTitle__text">
+                    {this.props.labelTitle}
```

Comment:
I would not use a regular prop here, I would use `this.props.children`. This way we can use this component event if we want to render text with new line for instance. Please take a look [here](https://mxstbr.blog/2017/02/react-children-deepdive/) for a explanation about children.